src/plugins/intel_cpu/src/emitters/plugin/riscv64/jit_eltwise_emitters.cpp

```diff
@@ -1514,6 +1514,56 @@ std::set<std::vector<element::Type>> jit_sigmoid_emitter::get_supported_precisio
     [[maybe_unused]] const std::shared_ptr<ov::Node>& node) {
     return {{element::f32}};
 }
+/// Greater///
+jit_greater_emitter::jit_greater_emitter(jit_generator* host, cpu_isa_t host_isa, const element::Type exec_prc)
+    : jit_emitter(host, host_isa, exec_prc) {}
+
+jit_greater_emitter::jit_greater_emitter(jit_generator* host, cpu_isa_t host_isa, const std::shared_ptr<ov::Node>& node)
+    : jit_emitter(host, host_isa, get_arithmetic_binary_exec_precision(node)) {}
+
+size_t jit_greater_emitter::get_inputs_num() const {
+    return 2;
+}
+
+size_t jit_greater_emitter::aux_vecs_count() const {
+    return 1;
+}
+
+size_t jit_greater_emitter::aux_fp_gprs_count() const {
+    return 0;
+}
+
+void jit_greater_emitter::emit_impl(const std::vector<size_t>& in_vec_idxs,
+                                    const std::vector<size_t>& out_vec_idxs) const {
+    if (host_isa_ == ov::intel_cpu::riscv64::cpu_isa_t::gv) {
+        emit_isa<ov::intel_cpu::riscv64::cpu_isa_t::gv>(in_vec_idxs, out_vec_idxs);
+    } else {
+        OPENVINO_THROW("Can't create jit eltwise kernel for GREATER");
+    }
+}
+
+template <ov::intel_cpu::riscv64::cpu_isa_t isa>
+void jit_greater_emitter::emit_isa(const std::vector<size_t>& in_vec_idxs,
+                                   const std::vector<size_t>& out_vec_idxs) const {
+    VReg lhs = VReg(in_vec_idxs[0]);
+    VReg rhs = VReg(in_vec_idxs[1]);
+    VReg dst = VReg(out_vec_idxs[0]);
+    VReg tmp = VReg(aux_vec_idxs[0]);
+    h->vmfgt_vv(mask_vreg(), lhs, rhs);
+    Xbyak_riscv::Reg reg_zero = h->t0;
```

Comment:
RISC-V has specific general purpose register `zero` (`x0`) which always contains zero. So no need to allocate one more register `t0` and push `0` to there. You can just [work](https://github.com/openvinotoolkit/openvino/blob/445f258a489645e0aae74735ae202458f7a2af61/src/plugins/intel_cpu/src/emitters/plugin/riscv64/jit_eltwise_emitters.cpp#L1037) with `zero` register :)